.github/workflows/run-checker-ci.yml

```diff
@@ -40,6 +40,7 @@ jobs:
           enable-trace enable-fips,
           no-ts,
           no-ui,
+          no-quic
```

Comment:
We ought to have an explicit `enable-quic` build somewhere too.
Doesn't have to be run checker, it could be on of the others.

Reply:
Ok. I added it back to the basic_gcc CI run and added a comment about it.